### DISCUSSION TITLE
Change of rootCA DigiCert to Sectigo/USERTrust: Remote IoT devices can no longer connect to api.github for OTA

### DISCUSSION THREAD

**Discussion by homepotager** [community/community#112103] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

We have some remote IoT devices that connect to api.github for OTA updates. These devices contain the rooCA certificate of DigiCert to connect to api.github. Since March 8, 2024, DigiCert is no longer used and the new rootCA is USERTrust. For the device, this is an untrusted source and does not connect any more.
Did I miss an announcement that this was going to happen? Was the change needed to switch from RSA to ECDSA, or another reason?
No need to say that this is a real bummer. I just want to understand the situation, make a strategy to fix this and prevent this in the future.
(Please understand that the fact that we are using Github for OTA - and that it may not be the best choice for OTA - is not the subject of this discussion.)
Best Regards.

**Comment by JanzenSAP** [comment]
https://github.blog/changelog/ does not list anything - it seems that this change was performed w/o prior announcements (bad style).

You were not the only one who got hit by that...
